packages/components/.storybook/web/main.ts

```diff
@@ -18,6 +18,7 @@ module.exports = {
   },
   core: {
     builder: 'webpack5',
+    disableWhatsNewNotifications: true,
```

Comment:
Not related to dependency updates but kept seeing this pop up as I was testing and decided to disable it.